program/c/src/oracle/oracle.h

```diff
@@ -27,12 +27,9 @@ extern "C" {
 #define PC_NUM_COMP_PYTHNET     128
 
 // PC_NUM_COMP - number of price components in use
-#ifdef PC_PYTHNET
 // Not whole PC_NUM_COMP_PYTHNET because of stack issues appearing in upd_aggregate()
 #define PC_NUM_COMP 64
-#else
-#define PC_NUM_COMP PC_NUM_COMP_SOLANA
```

Comment:
maybe delete `PC_NUM_COMP_SOLANA`

Reply:
I think this is still used in `PC_EXPECTED_PRICE_T_SIZE_PYTHNET`

Reply:
So perhaps remove it from there as well? :?

Reply:
Yep